### PR TITLE
CAPI-1210 : Add platform services to catalog during installation

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

CAPI-1210 : Add platform services to catalog during installation

### Description of the Change

1. Generate a service id and update all platform services with that.
2. Bootstrap job/script that will hold all the meta-data for the service id from step (1) and add it as a post-install step to dynamodb.

### Benefits

1. platform services use new structure
2. Service deployment workflows continue to work

### CODE REVIEW

**Comment by suryajak** [comment]
Please update the target to v1.3.1 branch

**Comment by SiniWilson** [comment]
@suryajak  I have updated the target to v1.3.1 branch

File: build-deploy-platform-services/Jenkinsfile [hunk 1]
```diff
@@ -433,3 +460,36 @@ def loadServiceConfigurationDataModule(buildURL,configLoader,role_arn, region, r
...
+			def service_data = parseJson(service_Object)
+			def data = service_data.Item.SERVICE_METADATA.M
+			def metadata = [:]
+			
+			for(item in data){
+				metadata[item.key] = item.value.S				
+			}			
+			return metadata
+		}
+	}
```
Comment (suryajak) [inline build-deploy-platform-services/Jenkinsfile]: region, domain, owner, service name are not part of service_metadata map, so needs to be added from outside.

region comes from configLoader where as the rest come from catalog

File: build-deploy-platform-services/Jenkinsfile [hunk 2]
```diff
@@ -276,6 +280,29 @@ def generateSwaggerEnv(service, env, deploymentNode, apiHostName, domain, servic
...
+	sh "sed -i -- 's/\${file(deployment-env.yml):providerTimeout}/" + config['providerTimeout'] + "/g' serverless.yml"
+	sh "sed -i -- 's/\${file(deployment-env.yml):eventScheduleRate}/" + config['eventScheduleRate'] + "/g' serverless.yml"
+	sh "sed -i -- 's/\${file(deployment-env.yml):eventScheduleEnable}/" + config['eventScheduleEnable'] + "/g' serverless.yml"
+	sh "sed -i -- 's/\${file(deployment-env.yml):securityGroupIds}/" + config['securityGroupIds'] + "/g' serverless.yml"
+	sh "sed -i -- 's/\${file(deployment-env.yml):subnetIds}/" + config['subnetIds'] + "/g' serverless.yml"
+	
+	if(config['artifact']){
+		sh "sed -i -- 's/\${file(deployment-env.yml):artifact}/" + config['artifact'] + "/g' serverless.yml"
+	}
+	if(config['mainClass']){
```
Comment (suryajak) [inline build-deploy-platform-services/Jenkinsfile]: can you make sure that these are being populated to catalog as part of java templates?

**Pull request merged**